spec/features/users/facebook_login_spec.rb

```diff
@@ -1,75 +1,53 @@
 require "rails_helper"
 
 describe "User logs in through facebook" do
-    before(:each) do
-        OmniAuth.config.mock_auth[:facebook] = nil
-
-        OmniAuth.config.mock_auth[:facebook] = OmniAuth::AuthHash.new(
-            {"provider"=>"facebook",
-            "uid"=>"100626065081257",
-            "info"=>
-            {"email"=>"sharon_vzvvxsf_moidustein@tfbnw.net",
-            "first_name"=>"Sharon",
-            "last_name"=>"Moidustein",
-            "image"=>"http://graph.facebook.com/v3.0/100626065081257/picture"},
-            "credentials"=>
-            {"token"=>
-                "EAAqHzoceqSsBAN90pOElp2umzOEvZBNkOqtMzTL13oZBQMLpaZAH1QfXYanI9hLEmOsYvDhJOpYzEZCObeSagX8DOfiHx24yysYPfuiCjNccgCzZBnZApziZC7kZCPDUTVpqsfmNMWACdlLJeUZCmBXJ3y2j54LrZBZBz5dGliPYmd6RtGo6bv52P7GZBtmY3VmTCrAZD",
-            "expires_at"=>1600736355,
```

Comment:
So the expiration didn't end being a factor at all?

Reply:
Not yet... who knows when this will actually expire...

Reply:
Not sure that it matters, since we are just using the name and email in the response and not reusing any of the other info that we receive back (especially just as a mock).